BDArmory/Radar/RadarUtils.cs

```diff
@@ -842,7 +842,7 @@ public static float GetStandoffJammingModifier(Vessel v, Competition.BDTeam team
 
             if ((BDArmorySettings.DEBUG_RADAR) && (modifiedSignature != signature)) Debug.Log("[BDArmory.RadarUtils]: Standoff Jamming: " + targetV.GetDisplayName() + " signature relative to " + v.GetDisplayName() + " modified from " + signature + " to " + modifiedSignature + "\n" + debugSOJ);
 
-            return modifiedSignature / signature;
+            return modifiedSignature / (signature == 0f ? 1f : signature);
```

Comment:
If `signature` is 0, shouldn't `GetStandoffJammingModifier()` just give a modifier of 1? This could also be checked at the start of the function.